setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
I'd like if we could move away from this.

The requirements (`demands` for example) mean that this can't easily be imported in `setup.py` without the outer virtualenv or system Python having them installed. I used the code mitsuhiko's [click](https://github.com/mitsuhiko/click/blob/master/setup.py#L1-L11) uses for reading the
`__version__` from the `__init__.py` file:

```python
import re
import ast
from setuptools import setup

_version_re = re.compile(r'__version__\s+=\s+(.*)')

with open('click/__init__.py', 'rb') as f:
    version = str(ast.literal_eval(_version_re.search(
        f.read().decode('utf-8')).group(1)))

setup(
    ...
    version=version,
    ...
)
```

So for uploading it or running `python setup.py test`/`python setup.py nosetests` you no longer need the packages' requirements installed.